Tools/clinic/clinic.py

```diff
@@ -27,6 +27,7 @@
 import types
 
 from types import *
+from typing import TypeVar, Callable
```

Comment:
Nit: typing.Callable is deprecated; better to use `collections.abc.Callable` in 3.12 :)

Reply:
TIL![^1] :)

[^1]: There's gonna be a lot of TIL's for me for the typed clinic project.